### PR TITLE
Add "call to action" for new activities

### PR DESCRIPTION
**Pull request by sergioalvz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,15 @@
 # Actividades de AsturiasHacking
 
+---
+
+<p align="center">
+  <b>
```
Comment (tasugo) [inline README.md]: Aren't the `---` and the line created by the paragraph a little redundant in the result markdown? I'd get rid of one of them

**Review by tasugo — CHANGES_REQUESTED** [review]
What do you think

File: README.md (hunk 1 above)
Comment (sergioalvz) [reply README.md]: Yeah, you are right! 👍

File: README.md [hunk 2]
```diff
@@ -1,5 +1,11 @@
 # Actividades de AsturiasHacking
 
+<p align="center">
```
Comment (tasugo) [inline README.md]: And now it looks less "important" xD

Maybe the other way around? I don't know anymore :(

**Review by tasugo — CHANGES_REQUESTED** [review]
:(

File: README.md (hunk 2 above)
Comment (sergioalvz) [reply README.md]: Another try! :)

**Review by tasugo — APPROVED** [review]
The best so far :+1:

**Pull request merged**